### PR TITLE
Add two missing titles in the settings bar

### LINKED ISSUE #1617: UI/UX: Not all icons in the settings bar have titles
When a user hovers over the icons in the settings bar some of them have a title and some don't. This is not great from a new user's perspective.

Screenshot from 2022-03-02 16-54-54

### PR DESCRIPTION
**Pull request by cypherhoodlum**
closes #1617.
Added missing titles for 'Bitcoin price' and 'Settings' in the settings bar.

### CODE REVIEW

**Comment by jonathancross** [comment]
Seems this has conflicts that must be resolved.

**Comment by k9ert** [comment]
Resolved! @moneymanolis Please merge! LGTM.

File: src/cryptoadvance/specter/templates/base.jinja [hunk 1]
```diff
@@ -70,7 +70,7 @@
 					<a class="settings-bar-btn" onclick="toggleHideSensitiveInfo()" style="padding: 0px 12px; cursor: pointer;" title="{{ 'Hide sensitive info' if not specter.hide_sensitive_info else 'Show sensitive info' }}">
 						<img src="{{ url_for('static', filename='img/hidden.svg' if specter.hide_sensitive_info else 'img/visible.svg') }}" style="width: 30px;" class="svg-white"/>
 					</a>
-					<a class="settings-bar-btn" href="{{ url_for('settings_endpoint.settings') }}" style="padding: 0px 7px;" title="Settings">
+					<a class="settings-bar-btn" href="{{ url_for('settings_endpoint.settings') }}" title="Settings" style="padding: 0px 7px;">
```
Comment (jonathancross) [inline src/cryptoadvance/specter/templates/base.jinja:73]: Seems

Comment (k9ert) [reply src/cryptoadvance/specter/templates/base.jinja:73]: I don't know what caused the conflict but the conflict was simply the order of the attributes.

**Pull request merged**